downstream/modules/aap-hardening/ref-security-variables-install-inventory.adoc

```diff
@@ -7,29 +7,36 @@
 
 [role="_abstract"]
 
-The installation inventory file defines the architecture of the {PlatformNameShort} infrastructure, and provides a number of variables that can be used to modify the initial configuration of the infrastructure components. For more information on the installer inventory, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Ansible Automation Platform Installation Guide].
+The installation inventory file defines the architecture of the {PlatformNameShort} infrastructure, and provides a number of variables that can be used to modify the initial configuration of the infrastructure components. For more information on the installer inventory, see link:{LinkInstallationGuide}.
 
 The following table lists a number of security-relevant variables and their recommended values for creating the installation inventory.
 
 .Security-relevant inventory variables
+[cols="25%,25%,25%,25%",options="header"]
 |===
-| *Variable* | *Recommended Value* | *Details*
-| `postgres_use_ssl` | true | The installer configures the installer-managed Postgres database to accept SSL-based connections when this variable is set.
+| *RPM Variable* | *Containerized Variable* | *Recommended Value* | *Details*
+| `postgres_use_ssl` | `postgre_disable_tls` |true | Determines if the connection between Ansible Automation Platform and the PostgreSQL database should use SSL/TLS. 
```

Comment:
we will need to QE this item, because it appears the two variables are applied with inverse logic.  that is, grammatically `postgres_use_ssl: true` and `postgres_dlsable_tls: false` would be equivalent, but we only have `true` as the value for the variable.  not sure how to best handle this in the table.